packages/calcite-components/src/components/color-picker/color-picker.e2e.ts

```diff
@@ -2157,7 +2161,7 @@ describe("calcite-color-picker", () => {
       const scope = await page.find(`calcite-color-picker >>> .${CSS.colorFieldScope}`);
 
       const initialStyle = await scope.getComputedStyle();
-      expect(initialStyle.left).toBe("0px");
+      expect(initialStyle.left).toBe("-0.5px");
```

Comment:
Per our conversation, would it be possible to avoid fractional pixel values? Not a deal breaker if not possible.